torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -480,19 +482,142 @@ void ONNXSetDynamicInputShape(
   }
 }
 
+bool HasSequenceTypeOutput(Node* node) {
+  if (node->kind() == ::c10::onnx::SplitToSequence ||
+      node->kind() == ::c10::onnx::SequenceInsert ||
+      node->kind() == ::c10::onnx::SequenceEmpty ||
+      node->kind() == ::c10::onnx::SequenceErase ||
+      node->kind() == ::c10::onnx::SequenceConstruct)
+    return true;
+  return false;
+}
+
+void ONNXUpdateTypeFromTensor(
+    Value* graph_output,
+    at::Tensor output,
```

Comment:
should not output be declared as  "const at::Tensor& output" we don't want to copy it every time and it is not changed

Reply:
Makes sense, updated. Thanks.